invenio_rdm_records/services/schemas/record.py

```diff
@@ -88,3 +90,28 @@ def default_nested(self, data, many, **kwargs):
             data["custom_fields"] = {}
 
         return data
+
+    def hide_tombstone_or_metadata(self, data):
+        """Hide tombstone info if the record isn't deleted and metadata if it is."""
+        is_deleted = data.get("deletion_status", {}).get("is_deleted", False)
+
+        if is_deleted:
+            keys_to_keep = ["tombstone", "deletion_status", "id", "pids", "links"]
+            for key in [k for k in data if k in keys_to_keep]:
+                data.pop(key, None)
+
+            pids = data.get("pids", {})
+            data["tombstone"]["identifiers"] = [
+                {"scheme": k, "identifier": v["identifier"]} for k, v in pids.items()
+            ]
```

Comment:
this basically just duplicates the information already available in `record['identifiers']`, massaged into a slightly more digestible format.
should we keep this around and/or throw `record['identifiers']` out, if the record is deleted?

Reply:
i guess the benefit of having the identifiers in the tombstone itself would be to have the `tombstone` information as self-contained as possible, so you can just send around that part of the whole record JSON and be fine with it.
but i also feel that this isn't exactly a compelling reason to have it around.

it's simple enough that we can re-add it at any point later, should we find that it would be actually beneficial to have.